### ISSUE TITLE
Is InstaPy Dead?

### ISSUE THREAD

**Issue by HashamGhuffary** [InstaPy/InstaPy#6715]
Hi everyone, it seems like this repo isn't being actively maintained anymore. 
It's sad to see such an amazing project end up like this.

Is anyone up for working on getting this project up and running again?

**Comment by aschreque** [comment]
I have the same question. I did some fix to make it works first for login and other small XPath trivial issue. 
Major change has been made by Instagram and feature of InstaPy are not working anymore.

I cannot post anything on Discord server in order to check who is doing what.

**Comment by bobe94** [comment]
I made small changes too in login and xpath. Now works again the auto like and auto-follow. 
But for the auto comment i really don't know how to do it. I get troubles on util.py  -->   getMediaData(query,browser,) function

**Comment by mian-amir** [comment]
any one tel me about instagram hacking commond pleas?

**Comment by mk1995** [comment]
> getMediaData

Yes the XHR retrieving from network is not working anymore as IG upgrade their API. Similarly get_shared_data and user data as well.

**Comment by uldiskk** [comment]
I probably shouldn't have wasted 3 hours making it start and only then deciding to check for existing issues...

**Comment by nhodges** [comment]
There's a fork where someone has resolved some of the xpath issues:

https://github.com/InstaPy2/InstaPy2

**Comment by sekoakb** [comment]
xpaths must be change. otherwise its timewaste..

**Comment by Torhamilton** [comment]
https://github.com/InstaPy2/InstaPy2 is buggy too.

**Comment by official-antique** [comment]
@Torhamilton how so? I plan to do a rewrite this year and would be grateful for recommendations

**Comment by torudo** [comment]
Already any updates?
